doc_user/UserManual.adoc

```diff
@@ -2079,7 +2081,7 @@ Open Hospital ships with the following set of reports:
 * OH23A1 - Open bills list: a report that shows the list of opened bills (pending bills) per period
```

Comment:
It would be nice not to use abbreviations (more professional IMHO) for (starting at line 2074):

* HMIS 33b - Weekly epidem. surveil.
* HMIS 33b - Weekly epidem. surveil. over 5 y.o.
* HMIS 33b - Weekly epidem. surveil. under 5 y.o.

Reply:
Done, thank you !